ios/sentry.properties

```diff
@@ -1,5 +1,5 @@
 defaults.url=https://sentry.io/
 defaults.org=pillar-project-worldwide-ltd
 defaults.project=rn-pillar-wallet
-auth.token=d63aa51d77ae45fea95a230c94aae4a910b5e8613b7745ca9a545f143728476f
+auth.token=d33bb0eb5dc4485c9c97450a328c39440542b6c11ead4c8099d20257b5513940
```

Comment:
Just checking, do we really want to change this?

Reply:
That's done by sentry wizard during the linking process, however I already asked on Pillar channel if this changes anything and no, besides, the tests were done on ios and android and everything works as expected

Reply:
That's done by sentry wizard during the linking process, however I already asked on Pillar channel if this changes anything and no, besides, the tests were done on ios and android and everything works as expected